README.md

```diff
@@ -746,7 +746,9 @@ API | Description | Auth | HTTPS | CORS |
 |---|---|---|---|---|
 | [arcsecond.io](https://api.arcsecond.io/) | Multiple astronomy data sources | No | Yes | Unknown |
 | [CORE](https://core.ac.uk/services#api) | Access the world's Open Access research papers | `apiKey` | Yes | Unknown |
+| [GBIF](http://api.gbif.org/v1/) | Global Biodiversity Information Facility | No | Yes | Yes |
 | [GBIF](https://www.gbif.org/developer/summary) | Global Biodiversity Information Facility | No | Yes | Yes |
+| [hubblesite](http://hubblesite.org/api/documentation/) | Space Telescope News Releases | No | No | Unknown |
```

Comment:
Can you please change `http` to `https`?

I can also see the API support the `HTTPS`. Can you please change `No` to `Yes`?